pkg/server/server.go

```diff
@@ -481,7 +481,7 @@ func (s *Server) Start(ctx context.Context) error {
 
 	ln, err := net.Listen("tcp", s.cfg.Addr)
 	if err != nil {
-		return err
+		return errors.Errorf("%s, use --port <port> to set the port", err)
```

Comment:
This is what `errors.Wrap` and `errors.Wrapf` are for, as in `errors.Wrap(err, "use --port <port> to set the port")`.